src/worker/exporters/amd_exporter.py

```diff
@@ -7,15 +7,16 @@
 
 import prometheus_client
 
+from rsmiBindings import rocmsmi, rsmi_status_t
+from RdcReader import RdcReader
+from rdc_bootstrap import *  # noqa: F403
+
 sys.path.extend([
     '/opt/rocm/libexec/rocm_smi',   # ROCm >=5.2
     '/opt/rocm/rocm_smi/bindings',  # ROCm <5.2
     '/opt/rocm/rdc/python_binding',
 ])
```

Comment:
Is this going to work with the path extensions after the import?

Reply:
Thanks for catching and addressed.